<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="en">
<context>
    <name>EditableGeoCsv</name>
    <message>
        <location filename="geocsv_plugin.py" line="58"/>
        <source>Add GeoCSV layer</source>
        <translation type="unfinished">Add GeoCSV layer</translation>
    </message>
    <message>
        <location filename="geocsv_plugin.py" line="65"/>
        <source>Editable GeoCSV</source>
        <translation type="unfinished">Editable GeoCSV</translation>
    </message>
</context>
<context>
    <name>GeoCsvNewController</name>
    <message>
        <location filename="geocsv_controller.py" line="106"/>
        <source>invalid file path</source>
        <translation type="unfinished">invalid file path</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="116"/>
        <source>unknown csvt attribute: {}</source>
        <translation type="unfinished">unknown csvt attribute: {}</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="118"/>
        <source>csv&lt;-&gt;csvt missmatch</source>
        <translation type="unfinished">csv&lt;-&gt;csvt missmatch</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="120"/>
        <source>csvt geometry type exception</source>
        <translation type="unfinished">csvt geometry type exception</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="122"/>
        <source>no csvt file found</source>
        <translation type="unfinished">no csvt file found</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="129"/>
        <source>error while loading csv</source>
        <translation type="unfinished">error while loading csv</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="166"/>
        <source>error in geometry selection</source>
        <translation type="unfinished">error in geometry selection</translation>
    </message>
</context>
<context>
    <name>GeoCsvReconnectController</name>
    <message>
        <location filename="geocsv_controller.py" line="215"/>
        <source>Couldn&apos;t automatically restore csv layer &quot;{}&quot;</source>
        <translation type="unfinished">Couldn&apos;t automatically restore csv layer &quot;{}&quot;</translation>
    </message>
</context>
<context>
    <name>VectorLayerSaveConflictController</name>
    <message>
        <location filename="geocsv_controller.py" line="264"/>
        <source>Save File</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="264"/>
        <source>Files (*.csv)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="271"/>
        <source>Invalid path</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="271"/>
        <source>An error occured while trying to save file on new location. Please try again.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>geocsvDialogNew</name>
    <message>
        <location filename="geocsv_dialog_new.ui" line="14"/>
        <source>GeoCSV Editor</source>
        <translation type="unfinished">Add new GeoCSV layer</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_conflict.ui" line="29"/>
        <source>Retry</source>
        <translation type="unfinished">Retry</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="68"/>
        <source>Cancel</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="geocsv_dialog_conflict.ui" line="43"/>
        <source>Save As</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="geocsv_dialog_conflict.ui" line="59"/>
        <source>Error: The csv file could not be saved. File Lock?</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="29"/>
        <source>Dateiname</source>
        <translation type="unfinished">Filename</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="46"/>
        <source>Durchsuchen...</source>
        <translation type="unfinished">search</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="78"/>
        <source>Ok</source>
        <translation type="unfinished">OK</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="122"/>
        <source>WKT Field</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="139"/>
        <source>Point (Easting/Northing)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="158"/>
        <source>WKT</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="203"/>
        <source>Easting Field</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="216"/>
        <source>Northing Field</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
